.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+    container: ubuntu:latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: |
+        apt-get update
+        DEBIAN_FRONTEND=noninteractive apt-get install -y \
+          git dpkg cmake gettext desktop-file-utils libdbusmenu-glib-dev libdbusmenu-gtk3-dev libgtk-4-dev \
+          meson \
+          ninja-build \
+          valac \
+          libgnome-menu-3-dev \
+          libxml2-utils \
+          libgtk-3-dev \
+          libgee-0.8-dev \
+          libbamf3-dev \
+          libwnck-3-dev \
+          bamfdaemon
+
+    - name: Build
+      run: |
+        meson setup --prefix="$(pwd)/usr" build
```

Comment:
This should be done slightly different, the setup should be for the final install location.

Ideally it should be:

```
  cd plank-reloaded
  meson setup --prefix=/usr build
  meson configure build -D enable-apport=false
  ninja -C build
  meson install --destdir="$(pwd)/usr" -C build
```

Reply:
Also there is a dist meson command. You can use

```
meson setup --prefix=/usr build
meson dist -C build
```